app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleSettingsFragment.kt

```diff
@@ -83,6 +85,10 @@ class LocaleSettingsFragment : Fragment() {
         })
     }
 
+    override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
+        TODO("Not yet implemented")
```

Comment:
Can we just return `false` here to indicate a menu item wasn't handled?

Reply:
Yes, I already did that for `SavedLoginsFragment`, I guess this slipped through.